### ISSUE TITLE
RFS Client exposes L2 data only

### ISSUE THREAD

**Issue by STBadman** [sunpy/radiospectra#203]
### Describe the feature

FIELDS/RFS L2 data is currently what is accessed directly by the RFS client always, but is uncalibrated, and the relevant cdf variable names can change between files

FIELDS/RFS L3 data is calibrated and includes an SFU at 1au and W/m^2/Hz product, whose cdf variable names are stable 

`
psp_fld_l3_rfs_hfr_PSD_FLUX 
psp_fld_l3_rfs_hfr_PSD_SFU
psp_fld_l3_rfs_lfr_PSD_FLUX
psp_fld_l3_rfs_lfr_PSD_SFU`

### Proposed solution

It would be better if the L3 data was the default but also if both are searchable, e..g through an a.Level Fido query

**Comment by viditxlogits** [comment]
@STBadman @hayesla For the L3 RFS CDFs, should RFSSpectrogram use PSD_SFU  or PSD_FLUX (W/m²/Hz) as the default spectrogram data variable? The L3 files provide both stable variables. 
If it should use PSD_SFU, how would be SFU represented , since astropy.units does not provide a built-in u.sfu unit?

**Comment by samaloney** [comment]
We could define a custom unit for SFU e.g.
```
SFU = u.def_unit('SFU', 1e-22 * u.W / (u.m**2 * u.Hz))
```
Are the arrays/values the same except for the unit?

**Comment by viditxlogits** [comment]
For L3 I'm reading the PSD_SFU variable directly, which the CDF provides normalised to 1 AU. The resulting array shape is (1545, 64) before transpose, and I'm only assigning the custom Astropy SFU unit to those values.
